### PR TITLE
[WIP] Add CFP committee page

### PR DESCRIPTION
**Pull request by skade**

### CODE REVIEW

File: _people/lislis.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+name: "Lisa Passing"
+groups:
+  - committee
+links:
+  - icon: github
+    link: https://github.com/lislis
+priority: 60
+---
+
+Lisa is a software developer and hobbyist game creator and creative
```
Comment (GuillaumeGomez) [inline _people/lislis.md]: Seems weird that the line is cut here whereas it's not in other descriptions, no?

Comment (skade) [reply _people/lislis.md]: Good catch, removed the newline.

File: _includes/cards/cfp-person.html [hunk 2]
```diff
@@ -0,0 +1,8 @@
+<div id="{{include.person.slug}}" class="card person">
+    <img src="/assets/people/{{include.person.slug}}.{% if include.person.jpg %}jpg{% else %}png{% endif %}" />
+    <div class="info">
+      <a href="{{include.person.url}}"><span class="name">{{include.person.name}}</span></a>
+      {% include links.html links=include.person.links %}
+      {{include.person.content}}
+    </div>
+  </div>
```
Comment (GuillaumeGomez) [inline _includes/cards/cfp-person.html:8]: nitpick: extra whitespaces at the end of this line.

File: _people/spastorino.md [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+  - committee
+links:
+  - icon: twitter
+    link: https://twitter.com/spastorino
+  - icon: github
+    link: https://github.com/spastorino
+priority: 60
+---
+
+
```
Comment (GuillaumeGomez) [inline _people/spastorino.md]: Extra line.

File: _people/lislis.md (hunk 1 above)
Comment (GuillaumeGomez) [inline _people/lislis.md]: I still see a newline here. :)

**Comment by flaki** [comment]
Just pushed some (dirty, but effective) CSS fixes in, also touched up some of the images so they come in the correct aspect ratio.

Is this ready to be pushed live/announced/tweeted @skade?

**Comment by flaki** [comment]
cfp committee

**Comment by skade** [comment]
Looks nice, here we go :).

**Pull request merged**